### DISCUSSION TITLE
Support for Managed SQL

### DISCUSSION THREAD

**Discussion by matthewkasiski** [ory/hydra#3467] [Ideas]
Trying to understand if there is support for managed SQL instances on the cloud service providers based upon this statement `"The SQL adapter supports PostgreSQL 9.6+, MySQL 5.7+ and SQLite. "` ( https://www.ory.sh/docs/hydra/self-hosted/dependencies-environment )

Examples such as:

**AWS**
- Amazon Aurora
- etc.

**Azure**
- Azure SQL
- Postgres via Cosmos DB
- etc.

**GCP**
- Cloud SQL
- etc.

**Comment by vinckr** [comment]
Hey @matthewkasiski 
I have no experience with these providers, but if they are fully compatible with SQL I think it should work. 
See here also: https://www.ory.sh/docs/self-hosted/deployment 

FWIW it would probably be cheaper to just get the [fully managed Ory Network](https://console.ory.sh/) which uses CockroachDB in the background, if you want to outsource database management. In my experience most who self-host want full control of the database.

**Comment by project0** [comment]
I just recently deployed hydra on AWS RDS Aurora Postgres, i dont think it uses any specific features what may not work in compatible managed databases.
